base/src/main/java/com/epmedu/animeal/base/theme/Color.kt

```diff
@@ -2,11 +2,23 @@ package com.epmedu.animeal.base.theme
 
 import androidx.compose.ui.graphics.Color
 
+val DisabledButtonColor = Color(0xFFD9DEE1)
```

Comment:
Should we add component variables to the color palette?

Reply:
> Should we add component variables to the color palette?

That are my variables, they should be probably in some another place, but be available for all UI components

Reply:
Will be done in scope of another ticket.